_data/navigation.yml

```
@@ -109,6 +109,8 @@
       link: spec/account-based-ticketing/
     - name: Non-Trip-Based Sales
       link: spec/non-trip-based-sale/
+    - name: fulfillment on wallets ad chip crads
```

```suggestion
    - name: Fulfillment on Wallets and Chip Cards
```

consistency and typo